CHANGELOG.md

```diff
@@ -1,7 +1,48 @@
-## Changelog (Current version: 1.3.1)
+## Changelog (Current version: 1.3.2)
 
 -----------------
 
+## 1.3.2 (2016 Apr 20)
+
+### Release Notes
+
+* __FIX__: although the previous version (1.3.1) fixed the exit code issue for `bitrise run`, the exit code was still not the right one in case of `bitrise trigger`. This version fixes the issue for bitrise trigger too, as well as we unified the handling codes of `run` and `trigger` as much as possible.
```

Comment:
Additionally, we now have integration tests (testing the exit codes) for both `bitrise run` and `bitrise trigger`.